### PR TITLE
webvitals scripts commit

### PR DESCRIPTION
**Pull request by mohammedluqmaan-deriv**
Adding base script for webpagetest API tests for staging.

### CODE REVIEW

File: webvitals/resources/login.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    capabilities["goog:loggingPrefs"] = {"performance": "ALL"}
+    chrome_options = Options()
+    chrome_options.add_argument('--headless')
+    # #    specify chrome driver path when testing locally     # #
+    #chrome_options.binary_location = "/Users/luqmaan/Desktop/chrome-mac-x64/"
+    driver = webdriver.Chrome(options=chrome_options)
+
+    try:
+        token_list = ''
+        DERIV_EMAIL = 'webvitals@deriv.com'
```
Comment (meeyun-deriv) [inline webvitals/resources/login.py:30]: lets put this in env var instead

File: webvitals/testcases/core_web_vitals.js [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    try {
+        console.log(`Test initiated --------> on URL: ${session_url.split('?')[0]}`);
+        const testResult = await new Promise((resolve, reject) => {
+            wpt.runTest(session_url, {
+                location: location,
+                connectivity: connectivity,
+                firstViewOnly: true,
+                label: tag_version,
+                device: device,
+                runs: 2,
```
Comment (meeyun-deriv) [inline webvitals/testcases/core_web_vitals.js:57]: is this the setting to get average from 2 runs?

**Review by meeyun-deriv — CHANGES_REQUESTED** [review]
do we also have a check in the image size?

File: webvitals/testcases/core_web_vitals.js (hunk 2 above)
Comment (mohammedluqmaan-deriv) [reply webvitals/testcases/core_web_vitals.js]: made it 3 was using 2 for test, Thanks

File: webvitals/resources/login.py (hunk 1 above)
Comment (mohammedluqmaan-deriv) [reply webvitals/resources/login.py]: Made the changes to fetch email from github env var.

**Comment by mohammedluqmaan-deriv** [comment]
> do we also have a check in the image size?

we have total images in first view metrics -                 "image_total": 3051,
but size related metrics i couldn't find i am currently looking into this there are megapixels i will check if there is a we determine using it.

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**